sdk/python/pyrightconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "venvPath": "venv",
+  "include": [
```

Comment:
I was looking at running type checkers across all of tests, but this is a quicker path to get something being tested